src/signals/incident-management/containers/IncidentDetail/IncidentDetail.tsx

```diff
@@ -301,7 +307,10 @@ const IncidentDetail = () => {
         <DetailContainer
           span={{ small: 1, medium: 2, big: 5, large: 7, xLarge: 7 }}
         >
-          <Detail attachments={state.attachments} context={state.context} />
+          <Detail
+            attachments={state?.attachments?.results}
```

Comment:
zitten attachments niet altijd in state? dan dan de ? weg.

Reply:
✅